lib/peddler/client.rb

```diff
@@ -18,6 +18,7 @@ class Client
     def_delegators :marketplace, :host, :encoding
 
     params('SellerId' => -> { merchant_id })
+    params('MWSAuthToken' => -> { auth_token })
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.